introduction.md

```diff
@@ -8,4 +8,6 @@ Some of my deadly skills include:
 * JavaScript
 * PHP
 * MySQL
-* Apache 
+* Apache
+* Linux
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?